docs/oss/getting-started/tutorial.md

```
@@ -1,267 +1,324 @@
+Then add `prerender: true` in the Rails view:
 
-- **[Heroku Deployment Guide](../deployment/heroku-deployment.md)** - Step-by-step Heroku deployment
-- **[General Deployment Guide](../deployment/README.md)** - Production deployment strategies for any platform
+```erb
+<%= react_component("Counter", props: @counter_props, prerender: true, auto_load_bundle: true) %>
+```
 
-These guides cover:
+> [!NOTE]
+> `prerender: true` needs a server bundle. For a first local SSR check, use `./bin/dev static` or run a production precompile. If you want the dev server to serve the prerender bundle, follow the [HMR guide](../building-features/hmr-and-hot-reloading-with-the-webpack-dev-server.md) and set `config.same_bundle_for_client_and_server = true` for that mode.
 
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)